index.bs

```diff
@@ -1324,9 +1324,10 @@ interface AudioContext : BaseAudioContext {
 </xmp>
 
 An {{AudioContext}} is said to be <dfn>allowed to start</dfn> if the user agent
-allows the context state to transition from "{{AudioContextState/suspended}}" to "{{AudioContextState/running}}". A user
-agent may require the actual audio production from an {{AudioContext}} to be
-<a>triggered by user activation</a> (as described in [[HTML]]).
+allows the context state to transition from "{{AudioContextState/suspended}}" to
+"{{AudioContextState/running}}". A user agent may delay this initial transition,
+to allow it only when the {{AudioContext}}'s [=relevant global object=] has been
+<a href="https://html.spec.whatwg.org/#tracking-user-activation">activated</a>.
```

Comment:
The explicit link doesn't look right? Do we need HTML to export something here to link to?

Reply:
Fixed by using bikeshed's syntax, since this is linking to a section.